app/assets/stylesheets/local/__variables.scss

```diff
@@ -12,6 +12,7 @@ $brand-red: #d8262e;
 $brand-yellow: #dbe341;
 //$brand-blue: #76c3d4;
 $brand-blue: #4bb0c7; // zack instructed us to use this slightly darker shade
+$brand-medium-blue: #4b81e8; // e.g. accept cookies button.
```

Comment:
waiting to hear back from Zack on if we should really use this non-brand-palette color,and if so if it's use is restricted to _just_ this location. (If the latter, we should name the variable $cookie-banner-blue or something to avoid confusion).